cm-vault/README.md

```diff
@@ -45,25 +45,24 @@ You can configurate custom variables in `common.tfvars`.
 
 ```
 # Path to the kubeconfig file to use for connecting kubernetes cluster. Default is "~/.kube/config"
-kubeconfig_path="~/.kube/config"
+kubeconfig_path = "~/.kube/config"
+
+# Cert-manger version to install. Default is "v1.8.2"
+cm_version = "v1.8.2"
```

Comment:
Note: we'll probably want to bump the default to v1.9.0 that was released last Friday https://github.com/cert-manager/cert-manager/releases

Reply:
Ok, I will do it.